src/js/components/Menu/stories/MenuCustomIcons.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Grommet, Box, Menu } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { Menu as HamburgerMenu, Close } from 'grommet-icons';
+import { deepMerge } from '../../../utils';
+
+const customTheme = deepMerge(grommet, {
+  menu: {
+    icons: {
+      down: HamburgerMenu,
+      up: Close,
+    },
+  },
+});
+
+const MenuCustomIcons = () => (
+  <Grommet theme={customTheme}>
+    <Box align="center" pad="large">
+      <Menu
+        dropProps={{
+          align: { top: 'bottom', left: 'left' },
+          elevation: 'xlarge',
+        }}
+        label="actions"
+        items={[
+          { label: 'Launch', onClick: () => {} },
+          { label: 'Abort', onClick: () => {} },
+          { label: 'Disabled', disabled: true },
+        ]}
+      />
+    </Box>
+  </Grommet>
+);
+
+storiesOf('Menu', module).add('Themed', () => (
```

Comment:
Can you please change the file name to Themed as well, so it will match the story.